js/common/objects/section.js

```diff
@@ -101,7 +101,7 @@ Section.prototype.onclick = function () {
 Section.prototype.setTime = function (time) {
     'use strict';
 
-    $(time).html(this.courseName)
+    $(time).html(this.courseName.substring(0,6) + ' (' + this.type + ')')

```

Comment:
The above calls to `substring` contain a space before the second parameter. I don't think we've outlined this in the wiki, but can you add this in here?